### PR TITLE
Style, grammar, and other friendly edits

### PR DESCRIPTION
**Pull request by dalanmiller**
Hey @kureikain! I thought I'd start the week by providing some edits for your first chapter and seeing what you thought. Have a look and let me know if you'd like me to continue!

### CODE REVIEW

File: manuscript/chapter1.txt [hunk 1]
```diff
@@ -7,73 +7,68 @@ improving the book content.
...
 
-Firsly, I'm not a RethinkDB expert at all. I'm just the average guy who loves
-programming and new technologies. To me, RethinkDB is very joyful to use.
+Firstly, I'm not a RethinkDB expert at all. I'm just an average guy who loves
+programming and new technologies. To me, RethinkDB is a pleasure to use.
 However, due to its age, there are not many books and documents about it comparing to other
-database systems. RethinkDB Doc/API document is very good. However, at the entry
-level, it can be hard to understand how to use an API or where to start, it's hard for people to
-jump direcly into API document and use it.
+database systems. While the RethinkDB cocumentation and API is very good, it can be hard to
```
Comment (danielmewes) [inline manuscript/chapter1.txt]: "hard to *know* where to start"?

File: manuscript/chapter1.txt [hunk 2]
```diff
@@ -81,44 +76,52 @@ If above example didn't convince you, then take this:
...
     r
-      .db('foodb')
+      .db('foodbase')
       .table('foods')
       .filter(r.row('created_at').year().eq(2011))
 
-This query is used to find all foods which were inserted in year **2011**. I
-don't even give equivalent SQL example, because it just cannot be as beautiful
-and verbose as above query.
+This query is finds all foods which were inserted in the year **2011**. I
```
Comment (danielmewes) [inline manuscript/chapter1.txt]: extra "is"

**Comment by dalanmiller** [comment]
Thanks @danielmewes

**Comment by kureikain** [comment]
@danielmewes Thanks and thanks a lot. I'm so sorry I didn't saw this before. My apology. I tured off Github notification :(. Please continue to help me. Appreciate a lot. I add you ask collaborator so you can have write access and merge directly

**Pull request merged**